### PR TITLE
Adding tests

### LINKED ISSUE #4: Add tests

### PR DESCRIPTION
**Pull request by rodolfocarobene**
This PR resolves #4 .

The tests currently have never been executed because qibosoq is not yet installed as package on the board.
In any case, I'm not sure if it will be possible to add automatic deployment of these, since they have to run on board... 
With qibolab it should be easier to run also "on qpu" tests

### CODE REVIEW

File: tests/test_rfsoc_server.py [hunk 1]
```diff
@@ -118,8 +118,8 @@ def test_execute_pulse_sequence_acquire(helpers):
         i, q = program.acquire(
             soc=soc, readouts_per_experiment=n_ro, load_pulses=True, progress=False, debug=False, average=False
         )
-        assert i.shape == [1, n_ro, 1000]
-        assert q.shape == [1, n_ro, 1000]
+        assert np.shape(i) == (1, n_ro, 1000)
```
[the remark this answers is not in the document]
Comment (rodolfocarobene) [reply tests/test_rfsoc_server.py:121]: Because the results of acquire are lists

[the remark this answers is not in the document]
Comment (AleCandido) [reply tests/test_rfsoc_server.py:121]: Ah... maybe we should return arrays. But if `qick` uses lists, return lists. Better to manipulate the output as little as possible.
(we might propose to use arrays directly in `qick`)

[the remark this answers is not in the document]
Comment (rodolfocarobene) [reply tests/test_rfsoc_server.py:121]: Yes Qick uses lists and, at least for averaged results, I just take the output as it is so I think it's better to use list here (for now). I do the conversion in arrays in qibolab, when I create the results objects

**Review by JavierSerranoGarcia — APPROVED** [review]

File: tests/test_rfsoc_server.py [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+                qubit=0,
+            )
+            sequence.add(p0)
+            start = p0.finish + 10
+        return sequence
+
+    def get_qubits():
+        channels = ChannelMap()
+        channels |= ChannelMap.from_names("L3-18_ro")  # readout (DAC)
+        channels |= ChannelMap.from_names("L2-RO")  # feedback (readout DAC)
```
Comment (JavierSerranoGarcia) [inline tests/test_rfsoc_server.py:62]:
```suggestion
        channels |= ChannelMap.from_names("L2-RO")  # feedback (readout ADC)
```

**Comment by rodolfocarobene** [comment]
This is outdated, I'll close this PR leaving the issue open

**Pull request closed without merging**